frontend/src/metabase/lib/urls/admin.ts

```diff
@@ -92,3 +92,7 @@ export function dataModelFieldFormatting(
   const tableUrl = dataModelTable(databaseId, schemaId, tableId);
   return `${tableUrl}/field/${fieldId}/formatting`;
 }
+
+export function adminLicense() {
+  return `/admin/settings/license`;
```

Comment:
I thought the linter would complain, but we reserve backticks for interpolated strings only.

```suggestion
  return "/admin/settings/license";
```

Reply:
Right! We can take try to add this to the linter. https://github.com/metabase/metabase/pull/59030/commits/574d53b68c340525afb4ae480e6bbaae77ae7ed0